sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Security/Ucfg/UcfgBuilder_Instructions.cs

```diff
@@ -44,12 +44,15 @@ public void Foo(string s)
             string a, b;
             a = s;                  // a := __id [ s ]
             a = ""boo""             // a := __id [ const ]
-            var x = new string[5];  // x := __id [ const ]
+            //var x = new string[5];
             int i;
             i = 5;                  // i := __id [ const ]
 
-            field = s;              // ignored
-            Class1.staticField = s; // ignored
+            field = s;              // this.field := __id [ s ]
+            this.field = s;         // this.field := __id [ s ]
+
+            staticField = s;        // Namespace.Class1.staticField := __id [ s ]

```

Comment:
Edge case - static fields of static classes referenced using the _using static MyNamespace.MyStaticClass;_ syntax